Networking/WebSockets/BuiltInWebSocket.cc

```diff
@@ -438,6 +438,8 @@ namespace litecore { namespace websocket {
             if (_usuallyFalse(n <= 0)) {
                 if (n < 0)
                     closeWithError(_socket->error());
+                else
+                    awaitWriteable();
```

Comment:
Looks like this is the only semantic change of this entire PR, right?

Reply:
Two main changes:
1. [TCPSocket] Ignored EWOULDBLOCK error in write().
2. [BuiltInWebSocket] Wait for the writable again when that happens.